app/utils/render-component.js

```diff
@@ -43,8 +44,15 @@ export function getRenderComponentFor(parent) {
 		componentInstance.renderer.appendTo(componentInstance, placeholderElement.parentNode);
 
 		scheduleOnce('afterRender', this, () => {
-			placeholderElement.parentNode.insertBefore(componentInstance.element, placeholderElement);
-			$(placeholderElement).remove();
+			if (componentInstance.element instanceof Node) {
+				placeholderElement.parentNode.insertBefore(componentInstance.element, placeholderElement);
+				$(placeholderElement).remove();
+			} else {
+				logEvent('render-component--element', {
+					componentName: name,
+					componentInstanceElement: JSON.stringify(componentInstance.element),
```

Comment:
selecting any `div` element in developer tools and executing `JSON.stringify($0)` returns `{}`. Maybe something like `componentInstance.element.outerHTML` would be better?